Dockerfile

```diff
@@ -1,46 +1,40 @@
-# VERSION 0.1
 # AUTHOR:           Clay Teeter <teeterc@gmail.com>, Nicholas Long <nicholas.long@nrel.gov>
 # DESCRIPTION:      Image with seed platform and dependencies running in development mode
 # TO_BUILD_AND_RUN: docker-compose build && docker-compose up
 
-# This Dockerfile has been updated to pull from our last known good build of SEED (v2.6.1).
-# Version 3.7.2-r2 of geos has introduced and incompatible library:
-#    https://pkgs.alpinelinux.org/package/edge/testing/x86_64/geos
-#FROM alpine:3.8
+FROM alpine:3.14
 
-# Start with 2.6.0. note that the source code will be removed and re-copied to this container. The
-# version of SEED here is used to load in the core system packages and dependencies.
-FROM seedplatform/seed:2.6.0
-
-# DO NOT UPGRADE until libgeos and shapely fix the connection.
-#RUN apk add --no-cache python \
-#        python3-dev \
-#        postgresql-dev \
-#        alpine-sdk \
-#        pcre \
-#        pcre-dev \
-#        libxslt-dev \
-#        linux-headers \
-#        libffi-dev \
-#        bash \
-#        bash-completion \
-#        npm \
-#        nginx && \
-#    apk add --no-cache --repository http://dl-cdn.alpinelinux.org/alpine/edge/main openssl && \
-#    apk add --no-cache --repository http://dl-3.alpinelinux.org/alpine/edge/testing/ geos gdal && \
-#    ln -sf /usr/bin/python3 /usr/bin/python && \
-#    python -m ensurepip && \
-#    rm -r /usr/lib/python*/ensurepip && \
-#    ln -sf /usr/bin/pip3 /usr/bin/pip && \
-#    pip install --upgrade pip setuptools && \
-#    pip install git+https://github.com/Supervisor/supervisor@837c159ae51f3 && \
-#    mkdir -p /var/log/supervisord/ && \
-#    rm -r /root/.cache && \
-#    addgroup -g 1000 uwsgi && \
-#    adduser -G uwsgi -H -u 1000 -S uwsgi && \
-#    mkdir -p /run/nginx && \
-#    echo "daemon off;" >> /etc/nginx/nginx.conf && \
-#    rm -f /etc/nginx/conf.d/default.conf
+RUN apk add --no-cache python3-dev \
+        postgresql-dev \
+        alpine-sdk \
+        pcre \
+        pcre-dev \
+        libxslt-dev \
+        linux-headers \
+        libffi-dev \
+        bash \
+        bash-completion \
+        npm \
+        nginx \
+        openssl-dev \
+        geos \
+        gdal \
+        gcc \
+        musl-dev \
+        cargo && \
```

Comment:
turned out we needed rust to install `cryptography` for python 🤷 
https://cryptography.io/en/latest/installation/#alpine